webservice/webservice.py

```diff
@@ -305,8 +306,14 @@ def get_manifest():
     """
     # Setup logging
     logger = logging.getLogger("dashboardService.webservice.get_manifest")
+    # Create query parameters.
     filters = request.args.get('filters', '{"file": {}}')
     logger.debug("Filters string is: {}".format(filters))
+
+    # NOTE: "format" here is a HTTP query parameter, not a Python command.
+    if not format:
```

Comment:
Because `format` has a default value on line 296, this line will never be true, will it?